openmetadata-ui/src/main/resources/ui/webpack.config.dev.js

```diff
@@ -19,7 +19,8 @@ const process = require('process');
 
 const outputPath = path.join(__dirname, 'build');
 const subPath = process.env.APP_SUB_PATH ?? '';
-const devServerTarget = process.env.DEV_SERVER_TARGET ?? 'http://localhost:8585/';
+const devServerTarget =
+  process.env.DEV_SERVER_TARGET ?? 'https://sandbox-beta.open-metadata.org/';
```

Comment:
Seems like it was committed by mistake. Can you please revert this?

Reply:
Reverted the changes.